src/containers/Home/Home.js

```diff
@@ -57,11 +58,22 @@ const Home = () => {
                 <Link to="/tasks">Tasks</Link>
               </StructuredListCell>
             </StructuredListRow>
+            {!!extensions.length && (
```

Comment:
`!!` ?

Reply:
`!!0 === false` (which React will then conveniently ignore in the output)
however if I left it as just `extensions.length` it'd display `0` on the page